pytensor/tensor/rewriting/linalg.py

```diff
@@ -131,6 +138,52 @@ def generic_solve_to_solve_triangular(fgraph, node):
                         ]
 
 
+@register_stabilize
```

Comment:
I want to make this a specialize only rewrite but for now it's in stabilize because pymc includes those before calling grad, and otherwise we still end up with messy unoptimized blockwise graphs